packages/palette/README.md

```diff
@@ -5,6 +5,30 @@ Artsy's Design System
 ## What is Palette?
 
 Palette is a collection of primitive, product-agnostic elements that help encapsulate Artsy's look and feel at base level. This project is intended to be used
-across our digital product portfoilo.
+across our digital product portfolio.
 
-Find out more at [github.com/artsy/palette](https://github.com/artsy/palette)
\ No newline at end of file
+Find out more at [github.com/artsy/palette](https://github.com/artsy/palette)
+
+### ⚠️ Development Gotchas
```

Comment:
```suggestion
### ⚠️ Don't Forget About iOS!
```

#minor - A more in-your-face title for this section. Not a requirement, but I'd be less likely to overlook a section named "Don't forget about iOS!" than I would "Development Gotchas".

Reply:
Ah good idea. This bug wasn't clear unfortunately so the louder the better... 

(We need to add a "Development" section to formal palette docs where this stuff can live. I'll add that to my todo list.)